x-pack/packages/kbn-ai-assistant/kibana.jsonc

```diff
@@ -1,5 +1,7 @@
 {
   "id": "@kbn/ai-assistant",
   "owner": "@elastic/search-kibana",
-  "type": "shared-browser"
+  "type": "shared-browser",
+  "group": "observability",
```

Comment:
the owner here is search-kibana but group is observability. Thoughts @sphilipse ?

Reply:
OK this group should be search. Short term we are owning it.

Reply:
```suggestion
  "group": "search",
```